src/activities/drawletters/drawletters.qrc~

```diff
@@ -0,0 +1,48 @@
+<RCC>
```

Comment:
I have committed the changes you have told. Can you please tell me , if I have made a mistake again or not ??. Thanks.